content/en/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -670,8 +670,18 @@ to launch an experiment from the command line:
 kubectl apply -f <your-path/your-experiment-config.yaml>
 ```
 
-For example, run the following command to launch an experiment using the
-random algorithm example:
+**Note that**:
+
+- If you deployed Katib as part of Kubeflow, you have to change Kubeflow namespace to your profile namespace.
+
+- (Optional) Katib's experiments don't work with
+  [Istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection).
+  If you installed Kubeflow using [Istio config](https://www.kubeflow.org/docs/started/k8s/kfctl-k8s-istio/),
+  you have to disable sidecar injection. To do that, specify annotation `sidecar.istio.io/inject: "false"`
+  in experiment's trial template. See how to do it for `Job`, `TFJob` or `PyTorchJob` in
+  [getting-started guide](http://localhost:1313/docs/components/hyperparameter-tuning/hyperparameter/#examples).
+
+Run the following command to launch an experiment using the random algorithm example:
 
 ```
```

Comment:
@andreyvelich If you have time, can you please add `shell` to three backticks whenever you see code blocks for command line examples? If you don't have time, don't worry. Thanks!

```suggestion
```shell
```

Reply:
> @andreyvelich If you have time, can you please add `shell` to three backticks whenever you see code blocks for command line examples? If you don't have time, don't worry. Thanks!

I'll do it, sure. Thanks!